src/app/components/Article.test.jsx

```diff
@@ -2,20 +2,85 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import Article from './Article';
 
-const HEADLINE = 'Article Headline';
+describe('Article', () => {
+  const testWrapper = (testTitle, testAssertion) => {
```

Comment:
By adding this `testWrapper`, the readability of the tests is reduced. 
Instead of reading the whole statement as a sentence `it('should return the fetch response'`  you get the abstraction of the wrapper instead: `testWrapper('should return the fetch response'...`. I'd suggest keeping the slightly more verbose version of the tests so that the readability of the `describe` & `it` blocks is retained.